packages/sdk/src/context.ts

```diff
@@ -0,0 +1,16 @@
+import { Value } from './Value';
+
+export interface Contextual<Self extends Contextual<Self>> {
+  getContext(): Context;
+  setContext(context: Context): void;
+  updateContextEntry<K extends string>(name: K, value: Context[K]): void;
+  removeContextEntry(name: string): void;
+  withContext(context: Context): Self;
+  clearContext(): void;
+}
+
+export interface Context extends Value.Struct {
+  openFeature?: Value.Struct & {
```

Comment:
Not blocking at all for this initial PR but just as a note: 
We want to declare any semantic context keys here asap in order to risk collisions later on (collisions such as customers defining context entries that we later plan as semantic contexts).